bazel/README.md

```diff
@@ -187,26 +187,29 @@ Envoy can produce backtraces on demand and from assertions and other fatal
 actions like segfaults. Where supported, stack traces will contain resolved
 symbols, though not include line numbers. On systems where absl::Symbolization is
 not supported, the stack traces written in the log or to stderr contain addresses rather
-than resolved symbols. The `tools/stack_decode.py` script exists to process the output
-and do symbol resolution including line numbers, to make the stack traces useful.
-Any log lines not relevant to the backtrace capability
-are passed through the script unchanged (it acts like a filter).
+than resolved symbols. If the symbols were resolved, the address is also included at
+the end of the line.
 
-The script runs in one of two modes. If passed no arguments it anticipates
-Envoy (or test) output on stdin. You can postprocess a log or pipe the output of
-an Envoy process. If passed some arguments it runs the arguments as a child
-process. This enables you to run a test with backtrace post processing. Bazel
-sandboxing must be disabled by specifying standalone execution. Example
-command line:
+The `tools/stack_decode.py` script exists to process the output and do additional symbol
+resolution including file names and line numbers. It requires the `addr2line` program be
+installed and in your path. Any log lines not relevant to the backtrace capability are
+passed through the script unchanged (it acts like a filter). File and line information
+is appended to the stack trace lines.
+
+The script runs in one of two modes. To process log input from stdin, pass `-s` as the first
```

Comment:
as a follow-up PR can you add doc that shows usage with "-s"? E.g. is this something I should try?
```
bazel test -c dbg //test/foo/bar_test --test_output=streamed | tools/stack_decode.py
```

I'm suggesting a follow-up because I don't want to further block your PR that adds the functionality on a doc tweak, especially with CI in its dysfunctional state.

Reply:
Will do it in a follow up doc-only PR. Your example here is not quite correct, both `-s` and the executable path need to be on the `stack_decode.py` command line, and tests output to stderr, not stdout. See a working example below.

```
bazel test -c dbg //test/server:backtrace_test --test_output=streamed |& tools/stack_decode.py -s bazel-bin/test/server/backtrace_test
```